rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.65"
```

Comment:
`clippy.toml` also needs update (also it might trigger some new lints)

Reply:
fixed!